test/JDBC/input/BABEL-808-vu-prepare.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE Employee (
```

Comment:
1. Please put the test scripts to input/functions and name it with function name, like we did for user_id
2. Please change all the table/view/procedure names to include the test title